docs/topics/media-pipeline.rst

```diff
@@ -197,6 +197,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_USE_SSL = False # or True (None by default)
     AWS_VERIFY = False # or True (None by default)
 
+
+Custom Amazon S3 headers in FileStorage
```

Comment:
Maybe you can remove the “in FileStorage” part, and describe the steps in a way that they work both for files and images?

Also, this should be a subheader. I believe you can use `'` for a lower-level header, e.g.

```
My header
'''''''''
```

Reply:
Done, I've added a more generalised subheader.